app/topics/show-topic/show-topic.component.html

```diff
@@ -3,7 +3,19 @@
   <md-card-title>
     <div class="title-wrapper">
       <div class="title"><h2>{{topic.title}}</h2></div>
-      <div class="status"><span>Status: {{ topic.status | translate}}</span></div>
+      <div class="status"><span *ngIf="showStatustoStudents">Status: {{ topic.status | translate}}</span>
+        <span *ngIf="!showStatustoStudents">
+    <md-radio-group class="radio-text" [(ngModel)]="topic.status" #status="ngModel" [disabled]=" showStatustoStudents "
+                    ngDefaultControl>
+      <md-radio-button value="InProgress">{{'in progress' | translate}}</md-radio-button>
+      <md-radio-button value="InReview">{{'in review' | translate}}</md-radio-button>
+      <md-radio-button value="Done">{{'done' | translate}}</md-radio-button>
+    </md-radio-group>
+    </span>
+        <span>
+           <md-icon class="save" (click)="saveStatus()" *ngIf="!showStatustoStudents">save</md-icon>
+        </span>
+      </div>
```

Comment:
Just show this directly when the User has the permission to change the status. Do not use the "Edit Content" button for that.